tests/testthat/test-krcomp.R

```diff
@@ -71,3 +71,50 @@ test_that("kr give similar results as SAS", {
   expect_equal(res$df, expected[2], tolerance = 1e-3)
   expect_equal(res$se, expected[1], tolerance = 1e-3)
 })
+
+# h_df_1d_kr
+test_that("h_df_1d_kr works as expected in the standard case", {
+  object_mmrm_kr <- get_mmrm_kr()
+  expect_snapshot(h_df_1d_kr(object_mmrm_kr, c(0, 1), TRUE))
```

Comment:
hm this is not ideal, as this will test for exact identity I think, can you check that and consider using `expect_snapshot_value()` instead or coding the expectations explicitly? (that would be better re: readability of the tests)